src/test/java/org/apache/commons/lang3/builder/TypedEqualsBuilderTest.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.lang3.builder;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TypedEqualsBuilderTest {
+
+    @Test
+    void test_complete_equals() {
+        TestObject testObject1 = new TestObject(1, 2);
+        TestObject testObject2 = new TestObject(1, 3);
+        TestObject testObject3 = new TestObject(1, 2);
+
+        assertEquals(testObject1, testObject1);
+        assertNotEquals(testObject1, testObject2);
+        assertNotEquals(testObject2, testObject1);
+
+        assertEquals(Boolean.TRUE,
```

Comment:
probably use assertTrue and assertFalse, unless perhaps there's some auto-boxing detail I'm missing

Reply:
It was done to make the test slightly more quick to read with of the syntax coloration.

Reply:
In general, do not assume devs are using IDEs. E.g. right now I'm reading this code in Github, and it's less clear than assertTrue

Reply:
GitHub has also syntax coloration. But yes let's change these asserts.